spec/features/merchant_items/show_spec.rb

```diff
@@ -26,3 +26,39 @@
     expect(page).to have_content(@item1.unit_price)
   end
 end
+
+RSpec.describe 'User story 33' do
+  before(:each) do
+    @merchant1 = Merchant.create!(name: "Pabu")
+    @merchant2 = Merchant.create!(name: "Loki")
+    @item1 = @merchant1.items.create!(name: "Ferret Food", description: "Yummy", unit_price: 20)
+    @item2 = @merchant1.items.create!(name: "Ferret Leash", description: "Long", unit_price: 20)
+    @item3 = @merchant2.items.create!(name: "Ferret Shampoo", description: "Tastey", unit_price: 20)
+  end
+
+  it 'has link to take to edit page' do
+    visit "/merchants/#{@merchant1.id}/items/#{@item1.id}"
```

Comment:
Can you do this by referencing the path with advanced routing?

Reply:
Yea, if possible that sounds like a great edit